README.md

```diff
@@ -2,13 +2,10 @@
 
 "Mathemagician" is a web app for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote.
 
```

Comment:
- [ ] Let's try to be detailed in our documentation as it can be of great benefit to users. You are yet to add the command for running the tests (`npm test`) in your documentation as requested previously.